test/pylib/cpp/common_cpp_conftest.py

```diff
@@ -71,13 +71,17 @@ def get_disabled_tests(config: dict, modes: list[str]) -> dict[str, set[str]]:
 
 def read_suite_config(directory: Path) -> dict[str, str]:
     """
-    Helper method that will return the configuration from the suite.yaml file
+    Helper method that returns the configuration from the test_config.yaml file.
+    It can be that there's no test_config.yaml that means there are no additional parameters for Scylla itself or no
+    filtering on what modes tests should be executed or skipped
     """
-    with open(directory / 'suite.yaml', 'r') as cfg_file:
-        cfg = yaml.safe_load(cfg_file.read())
-        if not isinstance(cfg, dict):
```

Comment:
Why did you drop this check, it seems applicable to test_config.yaml loading as well

Reply:
>  If there is an issue with yaml file, it will raise an exception.

Yes, but (correct me if I'm wrong) -- can there be a valid yaml file that is nonetheless safe_load()-ed into something that's not a dict? E.g. -- a list. Or a plain string.

...

Just checked. This file

```
- a
- b
```

successfully passes the yaml.safe_load() and renders a list. The `isinstance(x, dict)` returns False

Reply:
@xtrey , this concern is still valid

Reply:
done